### ISSUE TITLE
a.Ta is undefined (line graph not getting drawn)

### ISSUE THREAD

**Issue by edQua** [google/google-visualization-issues#2914]
When trying to draw a line graph it says a.Ta is undefined and the graph isn't getting drawn.
This happened in the last 16 hours.

**Comment by Birdie2020** [comment]
Bar charts also generate this error recently.

**Comment by valentinkrynin** [comment]
After update to 51 version of Loader.js all bar charts give an error message  "Cannot read property 'Do' of undefined× One or more participants failed to draw()"
<img width="549" alt="Знімок екрана 2021-06-24 о 11 59 43" src="https://user-images.githubusercontent.com/46224054/123234733-dddf5d80-d4e3-11eb-88cc-b2966e29aa0f.png">

**Comment by valentinkrynin** [comment]
For temporary solving that problem you have to load previous version of charts library. Change 'current' to '50'.
// google.charts.load('current', {'packages': packages});
google.charts.load('50', {'packages': packages});

**Comment by Birdie2020** [comment]
Thanks! That fixed it for the time being. Is there a chance that we can return to "current" some time (makes it easier to stay up to date)?

**Comment by snios** [comment]
Same bug for me. 
We where using "corechart" and "bar"
Error message: Cannot read property 'Do' of undefined
Solved it by changing to a frozen version. 51 = no go, 50 = works.

**Comment by hobbidhobbin** [comment]
Same issue. To solve either had to use version 50 or remove 'startup: true' from animation options

**Comment by dlaliberte** [comment]
Sorry for the inconvenience.  I suspect the problem that some people are having with v51 is the startup animation.  The charts and examples I am seeing are all working, however, and no one has yet pointed to an example that does not work, which would help in trying to debug it.

**Comment by valentinkrynin** [comment]
> Sorry for the inconvenience. I suspect the problem that some people are having with v51 is the startup animation. The charts and examples I am seeing are all working, however, and no one has yet pointed to an example that does not work, which would help in trying to debug it.

You're right, all my charts that have animation in their options give an error.
Example: animation: {startup: true, duration: 700, easing: inAndOut}

After removing animation chart draws well.

So the bug is in animation...

**Comment by ThHuberResearch** [comment]
@dlaliberte I have an example here that works with version 50, but does not work with the current version (51).

https://pastebin.com/BZRFAdUp

**Comment by dlaliberte** [comment]
Thanks @Madjura.  I see the 'startup': true option in your example.  I bet if you remove that, it will work in v51. No idea yet why it fails, but it has always been kind of a hack, so I am not too surprised that it broke again.

**Comment by clintlundmark** [comment]
Just noting that I am experiencing the same problem.  "Cannot read property 'Do' of undefined".  Removing "animation: {startup: true}" or switching to the older version 50 "google.charts.load('50', {'packages': packages});"is a work around.  The "bug" happens with multiple chart types.

**Comment by awakevwake** [comment]
This bug is only seen when you use the DataView and try to show the annotations data on the graph. 
var data=google.visualization.arrayToDataTable(dataArr);
//var view = new google.visualization.DataView(data);
//chart.draw(view, options);
chart.draw(data, options);
If you directly load the data the issue is not seen. Also, it disappears when you use animation.startup : false or if you lower down the version from current to 50.

**Comment by TonyKennah** [comment]
Great, I didn't have time to figure out what the issue was so just threw together a simple chart that worked.  Testing with version 50 and my old chart works again.
```
ChartLoader chartLoader = new ChartLoader(ChartPackage.CONTROLS);
chartLoader.setVersion("50");
```
